hasjob/models/jobpost.py

```diff
@@ -205,6 +204,12 @@ def get(cls, hashid):
         return cls.query.filter_by(hashid=hashid).one_or_none()
 
     @classmethod
+    def get_by_id(cls, jobpost_id, load_id_only=False):
```

Comment:
Since `JobPost.query.get(<id>)` loads all the columns and associated relationships, I added this method to enable a more efficient fetch. I've removed the method now.